src/app/pages/HomePage/index.tsx

```diff
@@ -68,10 +70,11 @@ const FooterStyled = styled(Box)(() => ({
 }))
 
 export const HomePage: FC = () => {
+  const apiStatusQuery = useGet()
   const [searchHasFocus, setSearchHasFocus] = useState(false)
   const theme = useTheme()
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
-
+  const isApiOffline = !apiStatusQuery.isLoading && !apiStatusQuery.isSuccess
```

Comment:
`const isApiOffline = apiStatusQuery.isFetched && !apiStatusQuery.isSuccess` behaves more correctly in cases where API is broken on first load